examples/example_HSC_DRP-Prod_start.yaml

```diff
@@ -0,0 +1,22 @@
+name: MUST_OVERRIDE
+parent_name: HSC_DRP-Prod
+spec_name: HSC_DRP-Prod_htcondor
+collection:
```

Comment:
As you noticed in pair coding, `collection` is spelled `collections`. Other than that, looks OK.